apps/e2e/helpers/fixtures.ts

```diff
@@ -24,16 +38,36 @@ export const testOrders = test.extend<OrderTestFixture>({
 	books: async ({ page }, use) => {
 		await page.goto(baseURL);
 
-		const books = [
-			{ isbn: "1234", authors: "author1", title: "title1", publisher: "pub1", price: 10 },
-			{ isbn: "4321", authors: "author2", title: "title2", publisher: "pub2", price: 20 }
-		];
 		const dbHandle = await getDbHandle(page);
 
 		// dbHandler
 		await dbHandle.evaluate(upsertBook, books[0]);
 		await dbHandle.evaluate(upsertBook, books[1]);
+		await dbHandle.evaluate(upsertBook, books[2]);
 
 		await use(books);
+	},
+	placedOrderLines: async ({ page }, use) => {
```

Comment:
I think we should update this to `placedOrders`, add way more lines and setup at least three orders: two the same supplier, one for a different supplier, and return the `{order & lines}[]`. It's as Ivan has suggested: to create more noise.